GiniVision/Classes/Core/Screens/Help/HelpMenuViewController.swift

```diff
@@ -135,9 +174,24 @@ extension HelpMenuViewController {
     }
     
     override public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        if let viewController = viewController(forRowWithId: items[indexPath.row].id) {
-            navigationController?.pushViewController(viewController, animated: true)
+        let item = items[indexPath.row]
+        let viewController = item.viewController
+        viewController.setupNavigationItem(usingResources: backToMenuButtonResource,
+                                           selector: #selector(back),
+                                           position: .left,
+                                           target: self)
+        
+        if let imageNoResultsViewController = viewController as? ImageAnalysisNoResultsViewController {
+            imageNoResultsViewController.didTapBottomButton = {
+                if let cameraViewController = (self.navigationController?
+                    .viewControllers
+                    .compactMap { $0 as? CameraViewController })?
+                    .first {
+                    _ = self.navigationController?.popToViewController(cameraViewController, animated: true)
+                }
+            }
         }
+        navigationController?.pushViewController(viewController, animated: true)
```

Comment:
You also assume a navigation controller setup. Again, something a coordinator should do.

Reply:
Fixed in the `HelpMenuViewControllerDelegate`